projects/vdk-core/src/vdk/internal/builtin_plugins/config/vdk_config.py

```diff
@@ -108,6 +108,9 @@ class EnvironmentVarsConfigPlugin:
     Configuration loaded from environment variables.
     """
 
+    def __normalize_key(self, key):
```

Comment:
Why not replace with a lambda?
```python
__normalize_key = lambda key: key.replace("-", "_").replace(".", "_").upper()
```

Reply:
Seems the same. The function style may be slightly more readable. I void lambda unless there are really really short